dts/arm/ambiq/ambiq_apollo3_blue.dtsi

```diff
@@ -268,6 +269,17 @@
 			ambiq,pwrcfg = <&pwrcfg 0x8 0x40>;
 		};
 
+		adc0: adc@50010000 {
+			reg = <0x50010000 0x400>;
+			interrupts = <18 0>;
+			interrupt-names = "ADC";
+			channel-count = <10>;
+			vref-mv = <1500>;
```

Comment:
@decsny should the **internal-vref-mv = <1500>;** be in this file or the board/overlay file?